Makefile

```diff
@@ -93,6 +93,11 @@ policy-controller:
 policy-tester:
 	CGO_ENABLED=0 go build -trimpath -ldflags "$(LDFLAGS)" -o $@ ./cmd/tester
 
+## Build localk8s binary
+.PHONY: localk8s
```

Comment:
Suggestion\nit: I would name it `local-dev` as we assume policy-controller just rans on k8s.